### DISCUSSION TITLE
customizing the cloud-init

### DISCUSSION THREAD

**Discussion by Baggypants** [cockpit-project/cockpit#19820] [Q&A]
The cloud init is amazing. Would it be possible to customize the user account creation to make the default user be in the wheel group so they can run commands as sudo? I'm assuming it's not already because of image compatibility. I'm also guessing it takes a template from the webform and creates an iso image in `/var/lib/libvirt/boot`?

**Comment by martinpitt** [comment]
This is a cockpit-machines related issue, and you already filed https://github.com/cockpit-project/cockpit-machines/issues/1377 , so handling it there.
